modules/signing_scriptworker/templates/passwords.json.erb

```diff
@@ -20,27 +20,6 @@
         ["mac-v2-signing13.srv.releng.mdc2.mozilla.com:9100", "<%= scope.function_secret(["signing_server_username"]) %>", "<%= scope.function_secret(["signing_server_nightly_password"]) %>", ["macapp"], "signing_server"],
         ["https://autograph-external.prod.autograph.services.mozaws.net", "<%= scope.function_secret(["autograph_mar_nightly_username"]) %>", "<%= scope.function_secret(["autograph_mar_nightly_password"]) %>", ["autograph_mar384", "autograph_hash_only_mar384"], "autograph"]
     ],
```

Comment:
The way I understood this is that this is duplicate definition with the one from `dep-passwords.json.erb` and the reason we have it is that we might want occasionally to still be able to sign on the production signing-servers with the `dep` key. If my understanding is right, this is not needed right now and we could clean this up.

Currently, machines using this template file are:
* [signing-linux-v1](https://tools.taskcluster.net/provisioners/scriptworker-prov-v1/worker-types/signing-linux-v1) from [here](https://github.com/mozilla-releng/build-puppet/blob/master/manifests/moco-nodes.pp#L275)
* [tb-signing-v1](https://tools.taskcluster.net/provisioners/scriptworker-prov-v1/worker-types/tb-signing-v1) from [here](https://github.com/mozilla-releng/build-puppet/blob/master/manifests/moco-nodes.pp#L299)

None of which, AFAIK, are using the `dep` definition whatsoever. 
CC'ing @escapewindow and @tomprince

Reply:
/me defers to Aki and Tom

Reply:
I believe so. I don't think it's harmful to keep it, but I also don't think it'll break anything to clean it up.

Reply:
Okay, will proceed with the cleaning. I'll keep an eye on things. Thanks!